packages/snaps-rpc-methods/src/restricted/manageState.ts

```diff
@@ -151,7 +126,7 @@ type GetEncryptionKeyArgs = {
  * from.
  * @returns The state encryption key.
  */
-export async function getEncryptionKey({
+export async function getEncryptionEntropy({
```

Comment:
Now that `snaps-rpc-methods` doesn't do encryption anymore, should we move this elsewhere?

Reply:
We could, where would you prefer it? I'm not sure where to move it since we still use the underlying entropy functions in other RPC methods 🤔

Reply:
Right, I forgot we still use it for entropy. Lets just keep it as is then.